apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.html

```diff
@@ -173,7 +173,7 @@ <h3>Address {{i+1}}</h3>
                                         </label>
                                         <input type="text" class="form-control" autocomplete="false"
                                             autocomplete="false" #gstNumberField
-                                            (keyup)="getStateCode(gst, states); checkGstNumValidation(gstNumberField);"
+                                            (keyup)="getStateCode(gst, states);" (blur)="checkGstNumValidation(gstNumberField);"
```

Comment:
Do keep the call to method `checkGstNumValidation` in `(keyup)` as the user can directly click submit after filling incorrect tax number which will take wrong code you can however keep `(blur)` also as an additional check